### PR TITLE
Added EFFECT parameter to ResponsiveBlast(s)

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: styles/responsive_styles.h [hunk 1]
```diff
@@ -59,14 +59,12 @@ using ResponsiveLightningBlockL =
...
       LayerFunctions<
-         Bump<Sin<Int<24>,Scale<BladeAngle<>,Int<14000>,Int<6000>>,Int<4000>>,
-              Sin<Int<17>,Scale<TwistAngle<>,Int<12000>,Int<9000>>,Int<8000>>>,
-         Bump<Sin<Int<27>,Scale<BladeAngle<>,Int<24000>,Int<10000>>,Int<8000>>,
-              Sin<Int<20>,Scale<TwistAngle<>,Int<10000>,Int<7000>>,Int<2000>>>,
-         Bump<Sin<Int<25>,Int<18000>,Scale<BladeAngle<24000,32768>,Int<22000>,Int<15000>>>,
-              Sin<Int<10>,Scale<TwistAngle<>,Int<6000>,Int<1000>>,Int<0>>>,
-         Bump<Sin<Int<31>,Scale<BladeAngle<22000,32768>,Int<30000>,Scale<BladeAngle<24000,32768>,Int<16000>,Int<10000>>>,Scale<BladeAngle<24000,32768>,Int<16000>,Int<8000>>>,
-              Sin<Int<18>,Scale<TwistAngle<>,Int<8000>,Int<2000>>,Int<0>>>>>, 
+         Bump<Scale<SlowNoise<Scale<BladeAngle<24000,32768>,Int<2100>,Int<1000>>>,Scale<BladeAngle<24000,32768>,Int<3000>,Int<10000>>,Int<16000>>,
```
Comment (profezzorn) [inline styles/responsive_styles.h:62]: Please indent better.

File: styles/responsive_styles.h [hunk 2]
```diff
@@ -88,22 +86,68 @@ using ResponsiveClashL =
...
-         class TR1 = TrInstant,
-         class TR2 = TrFade<200>,
+         int FADE = 400,
+         int SIZE = 100,
+         int SPEED = 400,
          int TOP = 28000,
          int BOTTOM = 8000,
-         int SIZE = 8000>
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
```
Comment (profezzorn) [inline styles/responsive_styles.h:103]: Move the > to the end of the previous line.

File: styles/responsive_styles.h [hunk 3]
```diff
@@ -88,22 +86,68 @@ using ResponsiveClashL =
...
+         int SIZE = 100,
+         int SPEED = 400,
+         int TOP = 28000,
+         int BOTTOM = 8000,
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
+using ResponsiveBlastWaveL =
+         MultiTransitionEffectL<
+                  TrWaveX<COLOR, Int<FADE>, Int<SIZE>, Int<SPEED>, Scale<BladeAngle<>, 
+                           Int<TOP>,
```
Comment (profezzorn) [inline styles/responsive_styles.h:127]: The way this is indented, it looks like an argument to TrWaveX, but it's an argument to Scale<>, please fix.

File: styles/responsive_styles.h [hunk 4]
```diff
@@ -88,22 +86,68 @@ using ResponsiveClashL =
...
+// TOP = uppermost Blast position limit, BOTTOM = lowermost Blast position limit, 32768 = tip, 0 = hilt
+
+template<class COLOR,
+         int FADE = 400,
+         int SIZE = 100,
+         int SPEED = 400,
+         int TOP = 28000,
+         int BOTTOM = 8000,
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
```
Comment (profezzorn) [inline styles/responsive_styles.h:123]: > goes at end of previous line

File: styles/responsive_styles.h [hunk 5]
```diff
@@ -59,14 +59,12 @@ using ResponsiveLightningBlockL =
...
-         Bump<Sin<Int<24>,Scale<BladeAngle<>,Int<14000>,Int<6000>>,Int<4000>>,
-              Sin<Int<17>,Scale<TwistAngle<>,Int<12000>,Int<9000>>,Int<8000>>>,
-         Bump<Sin<Int<27>,Scale<BladeAngle<>,Int<24000>,Int<10000>>,Int<8000>>,
-              Sin<Int<20>,Scale<TwistAngle<>,Int<10000>,Int<7000>>,Int<2000>>>,
-         Bump<Sin<Int<25>,Int<18000>,Scale<BladeAngle<24000,32768>,Int<22000>,Int<15000>>>,
-              Sin<Int<10>,Scale<TwistAngle<>,Int<6000>,Int<1000>>,Int<0>>>,
-         Bump<Sin<Int<31>,Scale<BladeAngle<22000,32768>,Int<30000>,Scale<BladeAngle<24000,32768>,Int<16000>,Int<10000>>>,Scale<BladeAngle<24000,32768>,Int<16000>,Int<8000>>>,
-              Sin<Int<18>,Scale<TwistAngle<>,Int<8000>,Int<2000>>,Int<0>>>>>, 
+         Bump<Scale<SlowNoise<Scale<BladeAngle<24000,32768>,Int<2100>,Int<1000>>>,Scale<BladeAngle<24000,32768>,Int<3000>,Int<10000>>,Int<16000>>,
+                  Scale<BrownNoiseF<Int<10>>,Scale<TwistAngle<>,Int<4000>,Int<10000>>,Scale<TwistAngle<>,Int<9000>,Int<14000>>>>,
```
Comment (profezzorn) [inline styles/responsive_styles.h:63]: fix indentation

File: styles/responsive_styles.h [hunk 6]
```diff
@@ -88,22 +86,68 @@ using ResponsiveClashL =
...
+         int FADE = 400,
+         int TOP = 28000,
+         int BOTTOM = 8000,
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
+using ResponsiveBlastFadeL =
+         MultiTransitionEffectL<
+                  TrConcat<TrInstant, 
+                           AlphaL<COLOR, Bump<Scale<BladeAngle<>, Int<TOP>, Int<BOTTOM>>, Int<SIZE>>>,
+                           TrFade<FADE>>,
```
Comment (profezzorn) [inline styles/responsive_styles.h:149]: fix indentation

File: styles/responsive_styles.h (hunk 1 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h:62]:
```suggestion
         Bump<Scale<SlowNoise<Scale<BladeAngle<24000,32768>,Int<2100>,Int<1000>>>,Scale<BladeAngle<24000,32768>,Int<3000>,Int<10000>>,Int<16000>>,
```

File: styles/responsive_styles.h (hunk 2 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
         BladeEffectType EFFECT = EFFECT_BLAST>
```

File: styles/responsive_styles.h (hunk 4 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
         BladeEffectType EFFECT = EFFECT_BLAST>
```

File: styles/responsive_styles.h (hunk 3 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
                  TrWaveX<COLOR, Int<FADE>, Int<SIZE>, Int<SPEED>, Scale<BladeAngle<>, Int<TOP>,  Int<BOTTOM>>>,
```

File: styles/responsive_styles.h [hunk 7]
```diff
@@ -88,22 +86,68 @@ using ResponsiveClashL =
...
+         int SPEED = 400,
+         int TOP = 28000,
+         int BOTTOM = 8000,
+         BladeEffectType EFFECT = EFFECT_BLAST
+         >
+using ResponsiveBlastWaveL =
+         MultiTransitionEffectL<
+                  TrWaveX<COLOR, Int<FADE>, Int<SIZE>, Int<SPEED>, Scale<BladeAngle<>, 
+                           Int<TOP>, 
+                           Int<BOTTOM>>>,
```
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
```

File: styles/responsive_styles.h (hunk 6 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
         MultiTransitionEffectL<TrConcat<TrInstant, 
                                                               AlphaL<COLOR, Bump<Scale<BladeAngle<>, Int<TOP>, Int<BOTTOM>>, Int<SIZE>>>,
                                                               TrFade<FADE>>,
```

File: styles/responsive_styles.h (hunk 3 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
using ResponsiveBlastWaveL =
         MultiTransitionEffectL<
                  TrWaveX<COLOR, Int<FADE>, Int<SIZE>, Int<SPEED>, Scale<BladeAngle<>, Int<TOP>, 
```

File: styles/responsive_styles.h (hunk 4 above)
Comment (fdarosa2663) [inline styles/responsive_styles.h]:
```suggestion
         BladeEffectType EFFECT = EFFECT_BLAST>
```

File: styles/responsive_styles.h (hunk 1 above)
Comment (profezzorn) [reply styles/responsive_styles.h:62]: These look the same to me. ALso, I've never seen these "suggested changes" before. I think you're doing it wrong.

**Comment by fdarosa2663** [comment]
Yeah, I think I messed something up too, I'm going to close this one and redo and resubmit, sorry for trouble I was trying to use the resolve but it didn't seem right.

**Pull request closed without merging**